addons/spectator/config.cpp

```diff
@@ -21,7 +21,9 @@ class CfgPatches {
 
 class CfgRespawnTemplates {
     class ADDON {
+        displayName = CSTRING(DisplayName);
         onPlayerKilled = QFUNC(respawnTemplate);
         onPlayerRespawn = QFUNC(respawnTemplate);
+        respawnTypes[] = {2,3};
     };
```

Comment:
There isn't afaik, but that has never stopped me. I just don't know which modes if any.